docs/getting-started/quick-start.md

```diff
@@ -6,9 +6,136 @@ sidebar_position: 2
 
 In this section we are providing a sample application that one can deploy and just use to play around.
 
-## K8S
+## Kubernetes
 
-_Here we provided a small sample on K8s_
+### Application
+
+An application is a process running within a container. Examples include a NodeJS Express application, a Ruby Rails application or a Spring Boot application.
+
+Create a NodeJS application:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: apps/v1 ############################### GROUP/VERSION
+kind: Deployment    ############################### RESOURCE
+
+metadata:
+  name: nodejs-rest-http-crud  #################### NAME
+  labels:
+    app: nodejs-rest-http-crud
+    runtime: nodejs
+    runtime-version: 14-ubi7
+
+spec:
+  selector:
+    matchLabels:
+      app: nodejs-rest-http-crud
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nodejs-rest-http-crud
+        deploymentconfig: nodejs-rest-http-crud
+    spec:
+      containers:
+      - name: nodejs-rest-http-crud
+        image: nodejs-rest-http-crud:latest
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+EOD
+```
+
+Application can be a podSpec based resource like "Deployment" or "DeploymentConfig" or a non podSepc based resource like a Custom Resource Definition.
+
+### Service
+
+Apply the following CatalogSource:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+    name: sample-db-operators
+    namespace: openshift-marketplace
+spec:
+    sourceType: grpc
+    image: quay.io/redhat-developer/sample-db-operators-olm:v1
+    displayName: Sample DB Operators
+EOD
+```
+
+Then navigate to the `Operators` -> `OperatorHub` in the OpenShift console and in the `Database` category select the `PostgreSQL Database operator` and install a beta version.
```

Comment:
You are in the Kubernetes section, aren't you?

Reply:
Updated this section.